### DISCUSSION TITLE
Is it possible to render a selection prompt in a layout

### DISCUSSION THREAD

**Discussion by kwabenadarkwa** [spectreconsole/spectre.console#1720] [Q&A] [answered]
<img width="1721" alt="Screenshot 2024-12-28 at 10 30 12 PM" src="https://github.com/user-attachments/assets/fc540bc9-24b6-457c-9a0d-343378dec8f0" />
for example in the area where the placeholder is, can I render a selection prompt in that section? 
how would I do that?

**Comment by patriksvensson** [comment, marked as answer]
No, that is not possible. This is by design.

**Reply by kwabenadarkwa** [reply]
Is there a work around that would allow me to acheive that?

**Reply by patriksvensson** [reply]
You would need to implement your own prompt as an IRenderable and host it in a live renderable somehow.
